core/src/main/java/tech/tablesaw/api/StringColumn.java

```diff
@@ -182,7 +182,25 @@ public String get(int rowIndex) {
 
   @Override
   public Table summary() {
-    return countByCategory();
+    Table summary = Table.create(this.name());
+    StringColumn measure = StringColumn.create("Measure");
+    StringColumn value = StringColumn.create("Value");
+    summary.addColumns(measure);
+    summary.addColumns(value);
+
+    measure.append("Count");
+    value.append(String.valueOf(size()));
+
+    measure.append("Unique");
+    value.append(String.valueOf(countUnique()));
+
+    Table countByCategory = countByCategory().sortDescendingOn("Count");
+    measure.append("Top");
+    value.append(countByCategory.stringColumn("Category").getString(0));
+
+    measure.append("Top Freq.");
```

Comment:
I'd probably call this just `"Freq"` for consistency with pandas and also because `"Top Freq."` sounds like it's the frequency of the `"Top"` item

Reply:
I'm not following this comment. This value is the frequency (number of occurrences) of the "Top" item, both here and in pandas.  
From the pandas docs - "The top is the most common value. The freq is the most common value's frequency."
I added the "Top"  because I find the naming in pandas confusing. It seemed better to be explicit and it makes it clearer that those values are a pair/go together when you view the result.

Reply:
Ok, looks good. Thanks for clarifying!